autoscheduler/frontend/src/theme.ts

```diff
@@ -39,3 +39,15 @@ const overrides: Overrides = {
 };
 
 export default createMuiTheme({ palette, overrides });
+
+const whiteButtonOverrides: Overrides = {
+  MuiIconButton: {
+    colorPrimary: {
+      color: '#fff',
+    },
+  },
+};
+
+export const whiteButtonTheme = createMuiTheme({
+  overrides: whiteButtonOverrides,
+});
```

Comment:
Check out what I did for this in #493, I'd prefer the new theme just has white as the primary color instead of using overrides since it's less lines and more flexible

Reply:
Yeah I like this way better